pkcs11/src/backend/object.rs

```diff
@@ -22,6 +22,7 @@ pub struct KeyRequirements {
     pub kind: Option<ObjectKind>,
     pub id: Option<String>,
     pub raw_id: Option<Vec<u8>>,
+    pub subject: Option<Vec<u8>>,
```

Comment:
Please rename to `cka_subject`, so that what it contains is clear.

Reply:
OK